tasks/unit-tests/utils_tests.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+
+from tasks.utils import clean_nested_paths
+
+
+class TestUtils(unittest.TestCase):
+    def test_clean_nested_paths(self):
+        paths = ["./pkg/utils/toto", "./pkg/utils/", "./pkg", "./toto/pkg", "./pkg/utils/tata"]
+        expected_paths = ["./pkg", "./toto/pkg"]
+        self.assertEqual(clean_nested_paths(paths), expected_paths)
+
```

Comment:
as per comment below, if you have `paths = ["./toto/titi", "./totoro", "./toto" ]` I think you have an issue